base/nsjail-docker/Dockerfile

```diff
@@ -23,6 +23,12 @@ RUN /usr/sbin/useradd --no-create-home -u 1000 user
 COPY --from=bin /usr/bin/nsjail /usr/bin/nsjail
 COPY --from=chroot /chroot /chroot
 
+# create a python3 venv and install ecdsa for the proof-of-work
+RUN apt-get update && apt-get -y install python3-venv
+RUN python3 -m venv /venv
+RUN bash -c "source /venv/bin/activate && pip3 install ecdsa"
+RUN mv /venv /chroot/
+
```

Comment:
I had some trouble with nsjail and the module search path. I.e. nsjail changes the PATH variable and python3 doesn't find the modules installed in /usr/local anymore.
Instead of adding it to the variable, I figured it's cleaner to use a virtualenv instead.

Reply:
Shouldn't we put this in the base chroot? Since it's part of the core infrastructure.

Reply:
The base/nsjail-docker is part of the core infrastructure too no?

Reply:
I mean, cached. So we don't need users to build this, but we can do it for them.

base/nsjail-docker is modifiable per CTF, and needs to be manually upgraded with new releases.

Reply:
If you want me to create more cached images, then I have a feature request #44 :)

Reply:
Haha ok :-)